WPMedia/arc-themes-components
# TMEDIA 687 separator

## Ticket

- [TMEDIA-687](https://arcpublishing.atlassian.net/browse/TMEDIA-687)

## Description

Takes in separator custom string via props. Set using ::after and scss per requirements 

## Acceptance Criteria

Provide a <Separator> component 

Default of dot separator 

Uses scss approach 

If a custom separator is provided, that is used

Have Scss file that adds the hooks to the Sass library

Have test coverage

Have storybook documentation

## Test Steps

1. Checkout branch - `git checkout TMEDIA-687-separator`
2. Run Storybook `npm run storybook`
3. Look for passing in separator string
4. Notice that the hook is still there for other customization than `content`
5. Check out the `separator` storybook documentation

## Author Checklist

_The author of the PR should fill out the following sections to ensure this PR is ready for review._


## Reviewer Checklist

_The reviewer of the PR should copy-paste this template into the review comments on review._
